### PR TITLE
Add in a no-op Go resolver

### PR DESCRIPTION
**Pull request by smukherj1**
Had to refactor the dependencies a lot. Will need to cut a release and
then update the README with the new instructions.

### CODE REVIEW

File: k8s/go/cmd/resolver/BUILD [hunk 1]
```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```
Comment (nlopezgi) [inline k8s/go/cmd/resolver/BUILD]: add licence header + visiblity

File: k8s/go/cmd/resolver/resolver.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package main
```
Comment (nlopezgi) [inline k8s/go/cmd/resolver/resolver.go]: license header

File: k8s/go/cmd/resolver/BUILD (hunk 1 above)
Comment (smukherj1) [reply k8s/go/cmd/resolver/BUILD]: Done

File: k8s/go/cmd/resolver/resolver.go (hunk 2 above)
Comment (smukherj1) [reply k8s/go/cmd/resolver/resolver.go]: Done

**Pull request merged**